L1Trigger/L1THGCalUtilities/python/hgcalTriggerChains.py

```diff
@@ -48,14 +48,14 @@ def register_chain(self, vfe, concentrator, backend1, backend2, selector='', ntu
             raise KeyError('{} not registered as ntuplizer'.format(ntuple))
         self.chain.append( (vfe, concentrator, backend1, backend2, selector, ntuple) )
 
-    def register_truth_chain(self, vfe, truth_prod, backend1, backend2, selector='', ntuple=''): 
+    def register_truth_chain(self, vfe, truth_prod, backend1, backend2='', selector='', ntuple=''): 
```

Comment:
`backend1` could also be set as default to `''`

Reply:
done